rasa_core/event_brokers/kafka_producer.py

```diff
@@ -0,0 +1,63 @@
+import logging
```

Comment:
this code is duplicated, let's keep the code in `broker.py` for now

Reply:
I've forgot to delete these files haha